### DISCUSSION TITLE
Dota 2 Classic for School Esports

### DISCUSSION THREAD

**Discussion by Le-Fluffy-Bunnie** [ValveSoftware/Dota2-Gameplay#3231] [Q&A]
Hi,

I am wanting to get some clarification on something. I am wanted to start an esports club at my school (in Australia) but due to networking restrictions League of Legends was a no go. Fortunately, Dota 2 supports offline LAN but in order to achieve this I had to use game version 6.84 from Dota2classic.com 

My question is, does Valve allow this or is this restricted by some legal barrier as it's not the most up to date version. I want to run this the correct way without doing something illegal. This way it can be a solution for all state schools here in Australia.

I was told to post here from Reddit. I have also already tried emailing Valve. No response.

**Comment by WilhelmDonnelly** [comment]
If you need the old version, only use the method of downloading the [grow a garden](https://growa-garden.io) depot from Valve's Steam server; do not share game files with each other.
